.github/workflows/check-plugin-docs.yml

```diff
@@ -0,0 +1,27 @@
+#
+# This GitHub action checks plugin repositories for valid docs.
+#
+# This provides a quick assessment on PRs of whether
+# there might be issues with docs in plugin repositories.
+#
+# This is intended to help debug Vercel build issues, which
+# may or may not be related to docs in plugin repositories.
+
+name: "website: Check plugin docs"
+on:
+  pull_request:
+    paths:
+      - "website/**"
```

Comment:
As soon as a plugin present in `website/data/docs-remote-plugins.json` releases a new latest version with invalid docs, all PRs with changes in `website/*` will start failing because of that. 

If it's possible, I think it makes sense to have it scheduled to regularly check remote docs and also on:
```
on:
  pull_request:
    paths:
      - "website/data/docs-remote-plugins.json"
```

Reply:
@sylviamoss I think the scheduled check would be a great idea - definitely open to adding that in a follow up PR, or maybe even here if it's straightforward (I need to brush up on my GH actions and see what's possible, will report back here once I have a rough idea 🏃) 

Re: changing the PR trigger for this check, I've been thinking of this "plugins" check as a kind of "canary" for the Vercel deploy check. If there are any invalid plugin docs, (eg due to a new latest version as in the scenario you mentioned), then all Vercel deploy previews, on all PRs that touch `website/*`, will start failing... but only folks with access (and time) to dive into the Vercel build logs will be able to figure out why. So I think it's preferable to have this check run every time we try to build the site, not just when the plugin config changes. That way, if the Vercel build fails, there's a quick indicator of whether or not it's related to invalid plugin docs.

Let me know if that makes sense - also more than happy to adjust, I feel like I might be missing something, but just wanted to make sure the intention was clear 🙇

Reply:
Oh nice it looks fairly simple to [add multiple event types](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#schedule), and looks like we already have a `scheduled` action running to check links, so this seems very very doable. Will update in a sec to run on both PRs and on a recurring basis 👍

Reply:
✅ All done! I've set the plugin docs check to run daily at `00:45`, to match the broken links checked. Happy to adjust if there's a different timing we'd prefer 👍

Reply:
Hey, this looks good! I think what you said makes totally sense 👍🏼